examples/v1/Map/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.24)
```

Comment:
Should this line even be in there?   While we're at it, maybe clean?   Same with other subdirs.